### DISCUSSION TITLE
Has the security concern found in the original repo been fixed?

### DISCUSSION THREAD

**Discussion by bishtawi** [nzbgetcom/nzbget#446] [Q&A]
Hi,

Glad to see that development of NZBGet has continued! I did a quick search and I didnt find anything so thought I would ask (apologies if this has already been covered). https://github.com/nzbget/nzbget/issues/833 was opened a few weeks before the original repo got archived. Looks like a potential security vulnerability was found.

With you guys resurrecting NZBGet, I am curious to know if you have reached out to either @benharvie or @underfl0w to learn more about what they have found and patched any potential vulnerabilities?

Thanks

**Comment by luckedea** [comment]
@bishtawi Thank you for bringing this up.
I'll reach out.

**Reply by bishtawi** [reply]
Hey, just following up on this as it's been over four months. Were you able to contact either @benharvie @underfl0w or @huntr-helper and fix the potential security issues they found?

I saw you created https://github.com/nzbgetcom/nzbget/pull/447 which is great btw 😀

Cheers

**Reply by luckedea** [reply]
Any known security issues (if any) have been resolved.
Every necessary precaution and action has been implemented.
